superset-frontend/src/explore/reducers/exploreReducer.js

```diff
@@ -219,6 +219,7 @@ export default function exploreReducer(state = {}, action) {
         slice: {
           ...state.slice,
           ...action.slice,
+          owners: action.slice.owners ?? null,
```

Comment:
This had to be explicitly added, as the action is missing the `owners` prop when all owners are missing